src/styles/BookInput.module.css

```diff
@@ -0,0 +1,71 @@
+.addNewBook {
+  font-family: Montserrat, 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
```

Comment:
- [ ] `normal` is the default styling for these styling. I can see that you are using the CSS given in Zeplin. however, by removing these lines you can save lines of code. kindly consider removing these styling in other instances as well.